tests/server.test.js

```diff
@@ -0,0 +1,23 @@
+const request = require('supertest');
+const mongoose = require('mongoose');
+const app = require('../server/index.js');
+
+afterAll(async () => {
+  await mongoose.disconnect();
+});
+
+it('endpoint responds with correct object', (done) => (
+  request(app)
+    .get('/api/product/00001')
+    .expect((response) => {
+      response.body.product_id = '00001';
+    })
+    .expect(200, done)
+));
+
+it('endpoint responds with status code 500 with wrong id', async (done) => {
+  const response = await request(app).get('/api/product/12345');
+
+  expect(response.status).toBe(500);
+  done();
```

Comment:
Would error code 404 be a more appropriate error message for a query with an incorrect/nonexistent id?

Reply:
Updated status code to 404